library/src/skins/default/src/sass/modules/tool/samigo/_samigo.scss

```diff
@@ -44,13 +44,17 @@
 	.samigo-question-callout {
 			border-left: 5px solid var(--tool-border-color);
 			margin: 2em 0;
-			padding: 1.5em 2.5em 1.5em 1.5em;
+			padding: 1px 2.5em 1.5em 1.5em;
 			input[type="checkbox"], input[type="radio"] {
 				margin-right: 0.25em;
 			}
 			label {
 				font-weight: normal;
 			}
+			img {
+			    margin-top: 1rem !important;
+			    margin-bottom: 1rem !important;
```

Comment:
would be great if you could avoid important by being more specific

Reply:
Understood. The rationale behind the use of "!important" here is that it's emulating what's used by a bootstrap class like "my-1" (which use "!important" liberally) without our having to insert such a bootstrap class for img elements across multiple JSP files (e.g., all the JSP files specific to question type). Isolating the margin to this SCSS file makes adjusting the margin for img elements within the samigo-question-callout div easier.

Reply:
bootstrap uses !important liberally to achieve consistency and reliability. when you use my-1 in bootstrap, you want to see that take effect instead of adjusting your local CSS. in our case, using !important is frowned upon because it makes debugging and adjusting more difficult.